test/configuration_test.py

```diff
@@ -5,7 +5,7 @@
 
 
 def test_init_valid():
-    Configuration(flags_configuration='{"flags": {}}')
+    Configuration(flags_configuration=b'{"flags": {}}')
```

Comment:
Since we're allowing bytes or a string, I imagine we should test both.

Also, let's have another test for initialization with a provided bandit configuration so we know it works!

Reply:
String is essentially deprecated and removed in 4.0.0